.github/workflows/build-and-push-assets.yml

```diff
@@ -80,6 +85,12 @@ jobs:
           echo "TAG_NAME=$(echo ${GITHUB_REF#refs/*/})" >> $GITHUB_ENV
           echo "COMPILE_SCRIPT=${{ inputs.COMPILE_SCRIPT_PROD }}" >> $GITHUB_ENV
 
+      - name: Setup SSH
```

Comment:
```suggestion
      - name: Set up SSH
```

Reply:
@tyrann0us There's another occurrency about `Setup` in https://github.com/inpsyde/reusable-workflows/blob/main/.github/workflows/build-and-push-assets.yml#L83 do you want we change it too? 
Btw, why changing it to `Set up`? isn't a common way to write `Setup` ?

Reply:
> @tyrann0us There's another occurrency about `Setup` in [https://github.com/inpsyde/reusable-workflows/blob/main/.github/workflows/build-and-push-assets.yml#L83](https://github.com/inpsyde/reusable-workflows/blob/main/.github/workflows/build-and-push-assets.yml?rgh-link-date=2022-09-07T07%3A30%3A14Z#L83) do you want we change it too? Btw, why changing it to `Set up`? isn't a common way to write `Setup` ?

@widoz, yes, my plan was to create a PR for fixing all the other typos. I didn't want to do it here, because it would "pollute" this PR too much IMO.

BTW, to _set up_ something is the verb, and _setup_ is the noun, see https://grammarist.com/spelling/set-up-vs-setup/. 🙂
However, the confusing thing is that the "Setup PHP" action does it intentionally wrong, see https://github.com/shivammathur/setup-php/pull/620. 🤷🏽‍♂️

Reply:
Done in [5749299](https://github.com/inpsyde/reusable-workflows/pull/12/commits/57492990bc5f3878c0ac607a15cab9cd62013a76)